pom.xml

```diff
@@ -144,6 +144,13 @@
                     <compilerArgument>-Xlint:deprecation</compilerArgument>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
```

Comment:
As per https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDBGDFI:
```
-source release
    Specifies the release of source code accepted. The release parameter can have the value 1.3, 1.4, 1.5, 6, 7, or 8. Use the value that corresponds to the value used when you compiled your code with the javac command.
```
apparently one could also specify the `<detectJavaApiLink>false</detectJavaApiLink>` setting to avoid this issue. Since it's working it's good enough for me.